projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/impala_lineage_plugin.py

```diff
@@ -111,8 +111,19 @@ def _is_query_have_lineage(query_statement: str) -> bool:
             if line.endswith("select 1 -- testing if connection is alive."):
                 return False  # Queries for keep alive connections does not provide meaningful lineage data
             if line.startswith(
-                    ("compute", "refresh", "describe", "set", "show", "explain", "invalidate", "revoke", "grant",
-                     "use")):
+                (
+                    "compute",
```

Comment:
How exhaustive is this list ?  Also can you sorted alphabetically ? It would be easier to see what's in. I decided to compare with https://docs.cloudera.com/documentation/enterprise/6/6.3/topics/impala_langref_sql.html#langref_sql and see if we should add something more but it's a bit of headache . So I will just ask you if you've checked the docs and evaluated all possible commands.

Reply:
List update, documentation noted